pkg/utils/netlink.go

```diff
@@ -35,6 +36,12 @@ type Netlink interface {
 	LinkSetDown(context.Context, netlink.Link) error
 	LinkSetMaster(context.Context, netlink.Link, netlink.Link) error
 	LinkSetNoMaster(context.Context, netlink.Link) error
+	LinkSetNsFd(context.Context, netlink.Link, int) error
```

Comment:
The PR descriptions says that we are making the tracer optional. However, we are adding new methods. Should it be a part of a separate commit?

Reply:
Hello @artek-koltun,

I will prefer to have those changes if it is possible to a single PR as it is easier for me to test the CNI [PR](https://github.com/opiproject/opi-gateway-evpn-cni/pull/14) against this one. Mainly these changes that are introduced in this PR have as purpose to make the netlink and FRR wrapper libraries to be reused by other components like the [EVPN GW CNI](https://github.com/opiproject/opi-gateway-evpn-cni/pull/14)

I can update the description of the PR if that is ok but I would prefer to not have multiple PRs as it is difficult to test the CNI against this. Hope that is ok.

Reply:
What if you test locally, then split and provide one PR and then another one?

Reply:
ok let me do the changes on this PR and test them. If they are working I will discard this PR and create two individual ones.

Reply:
Hello @artek-koltun,

I have made the changes with the noop tracer and tested them. They are working. I will create the two individuals PRs tommorow as we have discussed above

Reply:
Hello @artek-koltun,

I have splitted the changes in two PRs. The first one is this PR here and the second one is [this](https://github.com/opiproject/opi-evpn-bridge/pull/383)

Please review when you find some time.

Reply:
Could I ask you to update the storage bridges as well?